test/ctype_internal_test.c

```diff
@@ -13,6 +13,16 @@
 #include <ctype.h>
 #include <stdio.h>
 
+/*
+ * Even though the VMS C RTL claims to be C99 compatible, it's not entirely
+ * so far (C RTL version 8.4).  For the sake of these tests, we therefore
+ * define our own.
+ */
+#if defined(__VMS) && __CRTL_VER <= 80400000
+# include <string.h>
```

Comment:
For a local function in a test program, making it inline is a bit over the top, wouldn't you say?  ;-)